scss/components/lfd/agenda.scss

```diff
@@ -386,6 +389,24 @@
         background-color: map-get($configuration, agendaListDetailOverlayBackground);
       }
 
+      .file-holder {
+        .file-item {
+          .file-title {
+            color: $bodyTextColor;
+            font-weight: $bodyFontWeight;
+          }
+
+          .file-info {
```

Comment:
This entire `.file-info` declaration is unnecessary.

I mentioned "The metadata is rendered at 85% of the font size and without any change in font colour or opacity". Though, the mockup wasn't updated to reflect this. This means:

1. There's no change in color. No need for `color: rgba($bodyTextColor, 0.5);`
2. `.file-info { font-size: 0.85em; }` can be added to the LFD styles without having to use the theme files, which increases the compiled theme CSS file.